core/server/data/migrations/versions/2.28/2-add-db-backup-content-permission-to-roles.js

```diff
@@ -0,0 +1,46 @@
+const logging = require('../../../../lib/common/logging');
+const utils = require('../../../schema/fixtures/utils');
+
+const relationFixtures = {
```

Comment:
@allouis just wondering why you opted for explicit fixture here instead of using `utils.findModelFixtures('Permission', {object_type: resource});` method used in other migrations? Tbh like the explicitness of it but don't know if it's worth stepping from an already used pattern (no strong opinion here)

Reply:
Two reasons really 1) I wanted to be super explicit, to make it easier to review/read and 2) perf, I know it doesn't really matter but felt like an easy way to get rid of redundant db comms, and still have it be readable :relaxed: